app/population/middleware/set-regional-organisation-switcher.js

```diff
@@ -0,0 +1,13 @@
+async function setRegionalOrganisationSwitcher(req, res, next) {
```

Comment:
Is this still required? This looks like its related to the element beneath the organisation switcher that we discussed yesterday that seems to have been removed?

Reply:
Ah, I thought I'd dropped that code - good catch!

Reply:
Found and removed! :)